auth-web/src/components/auth/mixins/NextPageMixin.vue

```diff
@@ -91,18 +93,18 @@ export default class NextPageMixin extends Vue {
         // Redirect to dashboard otherwise
         orgName = encodeURIComponent(btoa(this.currentAccountSettings?.label))
         if (!this.userProfile?.userTerms?.isTermsOfUseAccepted) {
-          nextStep = Pages.USER_PROFILE_TERMS
+          nextStep = `/${Pages.USER_PROFILE_TERMS}`
```

Comment:
/ can be dangerous, isn't a relative path.

Reply:
it was like that before , instead of adding / at the end of return, just added in each line since bcRegistryDashoadUrl is a full URL